examples/counter_out/cont_gen_dig_pulse_train.py

```diff
@@ -0,0 +1,21 @@
+"""Example for continuously generating digital pulse train.
+
+This example demonstrates how to generate a continuous digital
+pulse train from a Counter Output Channel. The Frequency, Duty
+Cycle, and Idle State are all configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, Level
+
+with nidaqmx.Task() as task:
+    channel = task.co_channels.add_co_pulse_chan_freq(
+        "Dev1/ctr1", idle_state=Level.LOW, initial_delay=0.0, freq=100.0, duty_cycle=0.5
+    )
+    channel.co_pulse_term = "/Dev1/PFI13"
```

Comment:
You don't need to set this. Each ctrN channel has a different default PFI output.

Reply:
I will keep it to be consistent with CI examples.